src/python/pants/option/options_test.py

```diff
@@ -265,149 +275,156 @@ def test_arg_scoping(self):
     self.assertEqual(False, options.for_scope('test.junit').verbose)
 
     # Test list-typed option.
-    options = self._parse('./pants', config={'DEFAULT': {'y': ['88', '-99']}})
-    self.assertEqual([88, -99], options.for_global_scope().y)
-
-    options = self._parse('./pants --y=5 --y=-6 --y=77',
-                          config={'DEFAULT': {'y': ['88', '-99']}})
-    self.assertEqual([88, -99, 5, -6, 77], options.for_global_scope().y)
-
-    options = self._parse('./pants')
-    self.assertEqual([], options.for_global_scope().y)
-
-    options = self._parse('./pants ', env={'PANTS_CONFIG_OVERRIDE': "['123','456']"})
-    self.assertEqual(['123', '456'], options.for_global_scope().config_override)
-
-    options = self._parse('./pants ', env={'PANTS_CONFIG_OVERRIDE': "['']"})
-    self.assertEqual([''], options.for_global_scope().config_override)
-
-    options = self._parse('./pants --listy=\'[1, 2]\'',
-                          config={'DEFAULT': {'listy': '[3, 4]'}})
-    self.assertEqual([1, 2], options.for_global_scope().listy)
+    global_options = self._parse(
+      './pants', config={'DEFAULT': {'y': ['88', '-99']}}
+    ).for_global_scope()
```

Comment:
Several places in this file use this new idiom to define `global_options`. I found it makes it easier to read the assertion statement. Nothing else changed